source/SpinalHDL/Data types/Int.rst

```diff
@@ -396,13 +396,13 @@ Misc
      - Return the range (0 to x.high)
      - Range
    * - x.minValue
-     - Lowest value that x can take on (e.g. 0 for UInt)
+     - Lowest value of x (e.g. 0 for UInt)
```

Comment:
+= possible

for consistency with valueRange

Reply:
done